aws/resource_aws_eks_node_group.go

```diff
@@ -323,29 +324,29 @@ func resourceAwsEksNodeGroupCreate(d *schema.ResourceData, meta interface{}) err
 	_, err := conn.CreateNodegroup(input)
 
 	if err != nil {
-		return fmt.Errorf("error creating EKS Node Group (%s): %w", id, err)
+		return diag.Errorf("error creating EKS Node Group (%s)", id, err)
```

Comment:
Still need to pass `error`, just can't use `%w`.

```suggestion
		return diag.Errorf("error creating EKS Node Group (%s): %s", id, err)
```